packages/@uppy/provider-views/src/Item/components/ItemIcon.js

```diff
@@ -18,7 +18,7 @@ function FolderIcon () {
 
 function VideoIcon () {
   return (
-    <svg aria-hidden="true" focusable="false" viewBox="0 0 58 58">
+    <svg aria-hidden="true" focusable="false" style={{ width: 16, marginRight: 4 }} viewBox="0 0 58 58">
```

Comment:
@arturi does this look good to you?

Reply:
this should be fine—the other icons use similar inline styles